src/main/java/seedu/trackermon/model/show/Rating.java

```diff
@@ -0,0 +1,75 @@
+package seedu.trackermon.model.show;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.trackermon.commons.util.AppUtil.checkArgument;
+
+
+public class Rating {
+
+    public static final String INVALID_RATING = "Rating should be between 0 to 5";
+
+    public static final Integer FULL_RATING = 5;
+
+    public final Integer rating;
```

Comment:
Does the rating type need to be an `Integer` class instead of just being `int`?

Reply:
Hmmm good point! I will amend it!